misc/snapshotter/nydus-snapshotter.fscache.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=nydus snapshotter
+After=network.target
+Before=containerd.service
+
+[Service]
+Type=simple
+Environment=HOME=/root
+ExecStart=/usr/local/bin/containerd-nydus-grpc --fs-driver fscache --daemon-mode shared --config-path /etc/nydus/config.json
```

Comment:
Can users install both `fusedev` and `fscache`? If so, we'd better distinguish the config file.